requirements.txt

```diff
@@ -1,2 +1,3 @@
 requests[security]
-tldextract
\ No newline at end of file
+tldextract
+dreamhostapi
```

Comment:
can you move the `dreamhostapi` dependency to the `optional-requirements.txt` file?